### ISSUE TITLE
Neptune 2S Firmware issues

### ISSUE THREAD

**Issue by oinosme** [oinosme/elegoo-neptune2-2s-guide#1]
I've used a different txt and the original files provided by https://github.com/mlee12382/Elegoo-Neptune-Firmware/blob/main/1.%20Auto_leveling-20210903.zip

However after flashing there are no images for the icons in the display of the Neptune.

**Comment by oinosme** [comment]
I contacted Elegoo and they sent me the firmware. But after flashing I still have the same issues.
I checked wiring and everything seems ok, so something else is going on.

![IMG_0570](https://user-images.githubusercontent.com/87452236/146101375-c3464fb2-b3ef-45ea-af18-fe86ba10da99.jpg)
![IMG_0569](https://user-images.githubusercontent.com/87452236/146101383-645ac6c4-bc1b-4649-909d-d961be2d74dd.jpg)

**Comment by mlee12382** [comment]
Could it be a bad display cable? maybe there's a broken wire? try wiggling the wire around at either end and along the length while it's on and see if anything changes.

**Comment by oinosme** [comment]
I tried that but nothing. When flashing it all the display looks good within the flashing process: https://mega.nz/file/VswkGTLT#j7CG_Cx3mw3OzNpxqRUBZKuBo3jWO3BW7VSwAW-VaKM

It's interesting because text works fine but not images.

Elegoo suggested to flash only the pictures folder but didn't work either. 

First time I flashed before I had issues I had not unplugged it yet, not that I can remember. Once I had the issues I thought it was maybe like other printers that need the display off, hence I opened and disconnected the display. That didn't do it either. Maybe first time I flashed something got corrupted that isn't in the usual flash images. Will wait for more suggestions from Elegoo.

**Comment by oinosme** [comment]
I managed to get it working and it's very interesting. Elegoo after looking at my videos and logs suggested that because I was in macOS that it was probably doing something wrong when extracting the files from the provided zip. And that I should try Windows or something else.

So I tried the following:

1. Extract the files via command line in macOS. Reformat the card. Copy files = FAIL
2. Download a different unzip tool (Unzip RAR from unarchiver) = FAIL
3. Tried from windows = FAIL
4. Downloaded 7z but as I was reformatting from windows I noticed that the allocation unit size was 16, I changed this to 32. = SUCESS!!!

So I was wondering if I could further investigate and tried more things:

* Tried again but with only formatting from windows and unzip from mac fail
* Tried a bunch of combinations of many things to list and all failed. 

I was frustrated so I did what I did before but FAILED again. Some of the fails weren't the one reported in this issue, but it would never read the files from the card and just boot normally. Then I noticed that the card was empty and realized sometimes, windows wouldn't copy the files.

My last attempt was to format the card, without quick format, 32Kb Allocation unit size. Then unzip with 7z in windows. I copied all the files to mac and moved the files to the sd card from mac. And that did it.

So I'm more confused than even and don't understand what really is the underlying problem but at least it sort of works. 

I'll mark this issue as closed because the issue I thought was the files but in reality was the process somehow.
